src/models/models.js

```diff
@@ -1,4 +1,4 @@
-module.exports = {
-    User :  require('./User.js').default,
-    Product :  require('./Product.js').default,
-}
+export default {
```

Comment:
The `require` statement is not needed anymore.
You can export everything outside e.g. `export * from './user'` (this way every exported User's member, incl. default export, will be available outside models).